README.md

```
@@ -1,5 +1,44 @@
 # Dropshop
 
+<div align="center">
+
+# Dropshop
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)